backend/user_management/schemas.py

```
@@ -110,58 +115,42 @@
   
-class BillCreate(BaseModel):  
-    amount: float  
-    method: str  
-    trx_id: str  
-    status: str  
- 
-  
 class OrderCreate(BaseModel):  
-    order_items: List[OrderItemCreate]  
-  
+    order_items: List[OrderItemCreate]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)